src/pages/PrintPage.tsx

```diff
@@ -3,6 +3,8 @@ import { RouteComponentProps } from 'react-router-dom'
 import styled from 'styled-components'
 import QRCode from '../components/QRCode'
 import { findPartyById } from '../utils/find'
+import randomBase64 from '../utils/random'
+import encodeVotes from '../votecoding'
```

Comment:
nit: There was an ESLint rule to check for this (I'll have to look into what happened), but rule enforces that default export should have same name as file. Suggestion:
```js
import random from '../utils/random'
import encodeVotes from '../encodeVotes'
```

Reply:
`encodeVotes` sure, but I'd like to keep the `randomBase64` because I think that's more descriptive.